modules/social_features/social_swiftmail/social_swiftmail.install

```diff
@@ -90,3 +90,11 @@ function social_swiftmail_update_8003() {
     user_role_grant_permissions($role, ['use text format mail_html']);
   }
 }
+
+/**
+ * Add a new module setting.
+ */
+function social_swiftmail_update_9000(): void {
```

Comment:
Please change the update hook number to `11001`.

We should not have hook_update_N, where N is 8000, or 9000, as referenced in https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Extension%21module.api.php/function/hook_update_N/8.2.x

```
Note that the x000 number can never be used: the lowest update number that will be recognized and run for major version x is x001.
```